flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -53,6 +53,14 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
     }
 
+    private lazy var dateFormatter = {
+        let formatter = DateFormatter()
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
+        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSXXX"
+        return formatter
+    }()
```

Comment:
Would be possible to use https://github.com/getsentry/sentry-cocoa/blob/main/Sources/Sentry/NSDate%2BSentryExtras.m directly?